lib/generators/draft/resource/templates/specs/crud_spec.rb

```diff
@@ -28,7 +28,7 @@
   end
 
 <% attributes.each do |attribute| -%>
-  context "details page" do
+  describe "/<%= plural_table_name %>/[<%= singular_table_name.upcase %> ID]" do
```

Comment:
I haven't run it but would `singular_table_name.upcase` leave underscores? Might want to humanize it and then upcase?